app/konnect/dev-portal/register-and-create-app.md

```diff
@@ -0,0 +1,74 @@
+
+<details><summary>Key concepts</summary>
+
+{% capture konnect_concepts %}
+**Application:** An application enables developers to register and use APIs, whether they are your company’s own or those of a third-party provider. To facilitate diverse integrations, developers can create applications that consume multiple APIs from different providers, streamlining various services into a single offering.
+
+**Application credentials:** Credentials authorize developers to access an application. You can generate credentials automatically, or  manage them manually through an Identity Provider (IdP), like OIDC.
+
+**Developer platform:** The developer platform allows developers like you to locate, access, and consume APIs. Using the Dev Portal, you can browse and search API documentation, test API endpoints, and manage your own credentials. 
+{% endcapture %}
+
+{{ konnect_concepts | markdownify }}
+
+</details>
+
+## Register and create an application
+
+All developers must register through the {{site.konnect_short_name}} Dev Portal. A {{site.konnect_short_name}} admin can provide you with the correct registration URL.
+
+1. **Register or sign in to the developer platform**
+    
+    Navigate to Dev Portal and do one of the following:
+    * Register for access by navigating to the Dev Portal and creating an account.
+    * Sign in to the Dev Portal with single sign-on (SSO).
+
+    Once you're approved for access, you can create an application in Dev Portal.
+
+1. **Create an application**
+
+    Click **My Apps**. From the **My Apps** page, click **New App** and fill out the application information. 
+    
+    {:.note}
+    > **Note:** The Reference ID must be unique. If your organization is using the [OIDC](/konnect/dev-portal/applications/enable-app-reg#oidc-flow) flow for application registration, enter the ID of your third-party OAuth2 claim.
+
+1. **Add products to your application**
+
+    Click **Catalog**, click the product version you want to create an application for, and then click **Register**. In the registration pop-up, select the app you just created and click **Request Access**. Repeat this for as many product versions as you want to add to one application.
+
+    Each application can only work with one auth strategy. Dev Portal can contain products with different auth strategies, so keep this in mind when adding more than one product to your application.
+    
+    {:.note}
+    > **Note:** The Reference ID must be unique. If your organization is using the [OIDC](/konnect/dev-portal/applications/enable-app-reg#oidc-flow) flow for application registration, enter the ID of your third-party OAuth2 claim.
+
+1. **Create API credentials for your application**
+
+    You can generate credentials using one of the following methods:
+        1. To generate an API key from the Dev Portal, navigate to the **My Apps** page and click the application that you want to generate a credential for. 
```

Comment:
Formatting here is broken
![image](https://github.com/Kong/docs.konghq.com/assets/23319190/4b64cf37-0375-4772-8049-731c9894686a)